api/internal/controllers/analysis/analysis.go

```diff
@@ -147,11 +158,90 @@ func (c *Controller) hasDuplicatedHash(
 }
 
 func (c *Controller) publishInBroker(analysisID uuid.UUID) error {
-	response, err := c.GetAnalysis(analysisID)
+	res, err := c.GetAnalysis(analysisID)
 	if err != nil {
 		return err
 	}
 
 	return c.broker.Publish("", exchange.NewAnalysis,
-		exchange.Fanout, response.ToBytes())
+		exchange.Fanout, res.ToBytes())
+}
+
+//TODO:REMOVE ALL BELOW AFTER v2.10.0
+func (c *Controller) treatCompatibility(analysisEntity *analysis.Analysis) error {
+	if !c.existsDepecratedHashesSlice(analysisEntity.AnalysisVulnerabilities) {
+		return nil
+	}
+	hashSlice := make([]string, 0)
+
+	for i := range analysisEntity.AnalysisVulnerabilities {
+		hashSlice = append(hashSlice,
+			analysisEntity.AnalysisVulnerabilities[i].Vulnerability.DeprecatedHashes...)
```

Comment:
I think that this syntax is better to read?
```go
hashSlice = append(
	hashSlice,
        analysisEntity.AnalysisVulnerabilities[i].Vulnerability.DeprecatedHashes...,
)
```

Reply:
if i add one more line into the function it breaks the lint

Reply:
I prefer break the lint over had a bad code to read.